modules/org/infinispan/runtime/added/bin/launch.sh

```diff
@@ -55,6 +54,15 @@ generate_content() {
   fi
 }
 
+execute_cli() {
+  # Use the native CLI if present
+  if [[ -f "${ISPN_HOME}/cli" ]]; then
+    "${ISPN_HOME}/cli" "$@"
+  else
+    "${ISPN_HOME}/bin/cli.sh" "$@"
+  fi
+}
```

Comment:
We should probably check the return code of the process here and quit if the CLI execution fails. Without this, the server will continue to start when the dependencies can't be resolved.

Reply:
Fixed